### PR TITLE
[Fundamental]Fix Compliance Check

### PR DESCRIPTION
**Pull request by crazygao**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

# All Promptflow Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes].**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](../CONTRIBUTING.md).**
- [ ] **Create an issue and link to the pull request to get dedicated review from promptflow team. Learn more: [suggested workflow](../CONTRIBUTING.md#suggested-workflow).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### Testing Guidelines
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: scripts/compliance-check/user_exclusion.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <PoliCheckExclusions>
   <!-- All strings must be UPPER CASE -->
   <!--index-xxx.js is an auto-generated javascript file - skipped given it's not expected to be readable -->
-  <Exclusion Type="FileName">SRC\PROMPTFLOW\PROMPTFLOW\_SDK\_SERVING\STATIC\INDEX.JS</Exclusion>
+  <Exclusion Type="FileType">.JS</Exclusion>
```
Comment (zhengfeiwang) [inline scripts/compliance-check/user_exclusion.xml:4]: can we only ignore min.js? as .js should be readable

Comment (crazygao) [reply scripts/compliance-check/user_exclusion.xml:4]: let me try

**Review by zhengfeiwang — APPROVED** [review]

File: .github/pipelines/compliance_check.yml [hunk 2]
```diff
@@ -22,36 +22,28 @@ pool:
...
+    filePath: '$(sourceLocation)\scripts\compliance-check\Check-PolicheckScan.ps1'
     arguments: >
-      -policheckResult $(sourceLocation)/scripts/compliance-check/result.tsv
+      -policheckResult $(sourceLocation)\scripts\compliance-check\result.tsv
   displayName: 'Check result'
 
 - task: PublishPipelineArtifact@1
   condition: failed()
   inputs:
-    targetPath: '$(sourceLocation)/scripts/compliance-check/result.tsv'
```
Comment (D-W-) [inline .github/pipelines/compliance_check.yml:54]: Why changed this?

Comment (crazygao) [reply .github/pipelines/compliance_check.yml:54]: Just fix the path inside the pipeline like this
image

**Review by D-W- — APPROVED** [review]

**Pull request merged**